README.md

```diff
@@ -5,6 +5,11 @@ arcades of the 80s. It was meant to be developed as a collaborative
 programming exercise in a course on Open Source Systems taught to
 undergraduate CS students.
 
+
+## Building/Running with Nix
```

Comment:
It could be inside the Quick start section, following the existing pattern, e.g. "If you are using nix...".